sigs.yaml

```diff
@@ -2003,12 +2003,11 @@ sigs:
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/OWNERS
       meetings:
       - description: Testing Commons
-        day: Wednesday
+        day: Friday
         time: "07:30"
         tz: "PT (Pacific Time)"
         frequency: bi-weekly
-        url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
-        archive_url: https://docs.google.com/document/d/1TOC8vnmlkWw6HRNHoe5xSv5-qv7LelX6XK3UVCHuwb0/edit#heading=h.tnoevy5f439o
```

Comment:
AFAIK this needs to be flipped back to the way it was, the `archive_url` uri should link to meeting notes / agenda